aikau/src/test/resources/alfresco/documentlibrary/views/MediaFilmStripViewTest.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (C) 2005-2016 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * The purpose of this test is to ensure that keyboard accessibility is possible between the header and the 
+ * main table. It should be possible to use the tab/shift-tab keys to navigate along the headers (and the enter/space key
+ * to make requests for sorting) and then the cursor keys to navigate around the table itself.
+ * 
+ * @author Dave Draper
+ * @author Martin Doyle
+ */
+define(["alfresco/TestCommon",
+        "intern!object",
+        "intern/chai!assert"],
+       function(TestCommon, registerSuite, assert) {
+
+   registerSuite(function() {
+      var browser;
+
+      return {
+         name: "Film Strip View Tests (Media Playing)",
+
+         setup: function() {
+            browser = this.remote;
+            return TestCommon.loadTestWebScript(this.remote, "/MediaFilmStripView?autoPlay=true", "Film Strip View Tests (Media Playing)").end();
+         },
+         
+         beforeEach: function() {
+            browser.end();
+         },
+
+         // By default autoPlay is disabled, but the test page overrides the plugin configuration...
+         "Check that video is playing": function() {
+            return browser.findDisplayedByCssSelector("video")
+               .execute("return document.querySelector('video').paused")
+               .then(function(paused) {
+                  assert.isFalse(paused, "The video is not playing");
```

Comment:
None of these extra comments are needed. If the test fails you'll get "Check that video is playing" expected true to be false ... really don't believe that extra string is necessary (in the interests of brevity of code if it doesn't impact on meaning).